src/jobs/test-ts-update.yml

```diff
@@ -10,7 +10,7 @@ parameters:
   node_version:
     description: version of node to run tests against
     type: string
-    default: latest
+    default: '15.6.0'
```

Comment:
shouldn't this be 'latest' so it always runs on the latest version and not 15.6.0?

Reply:
Nice catch here! Forgot to change this file